scenarios/09_Hired_Swords.cfg

```diff
@@ -128,13 +128,16 @@
         facing=ne
         side=1
         recruit=Dune Soldier,Dune Rover,Dune Burner
-        {GOLD 180 160 140}
-        {INCOME 5 4 3}
+        {GOLD4 180 160 140 120}
+        {INCOME4 5 4 3 2}
         [modifications]
             {TRAIT_LOYAL}
 #ifdef EASY
             {TRAIT_STRONG}
 #endif
+#ifdef NIGHTMARE
```

Comment:
No dim please.

Reply:
removed in [086c1d2](https://github.com/nemaara/A_New_Order/pull/53/commits/086c1d25c13ea2fc5549ae746b68eb2476cc283a).